docs/guide/plugins.md

```diff
@@ -0,0 +1,38 @@
+# Plugins
+
+Plugins are now supported in the application.
```

Comment:
I would add some more descriptive comment here. Something like: Plugins are nodejs applications that can be initegrated with the application in order to add some features. They have access to all the clients and express instance